internal/command/action.go

```diff
@@ -200,3 +212,26 @@ func returnStatusOfError(err error) (int, error) {
 	}
 	return 0, err
 }
+
+//Prints a list of all provider versions
+func printProviderVersions()(error){
+	providers, err := prov.GetAllProviders()
+
```

Comment:
Stray newline - errors are usually checked in the same logical block as the invocation